docs/standard/serialization/write-custom-serializer-deserializer.md

```diff
@@ -25,6 +28,7 @@ The following sections show how to use these tools for reading and writing JSON.
 The following example shows how to use the <xref:System.Text.Json.JsonDocument> class for random access to data in a JSON string:
 
 :::code language="csharp" source="snippets/system-text-json-how-to/csharp/JsonDocumentDataAccess.cs" id="AverageGrades1":::
+:::code language="vb" source="snippets/system-text-json-how-to/vb/JsonDocumentDataAccess.vb" id="AverageGrades1":::
```

Comment:
Hmm I cant see the VB code in this section - https://review.docs.microsoft.com/en-us/dotnet/standard/serialization/write-custom-serializer-deserializer?branch=pr-en-us-22414#use-jsondocument-for-access-to-data.

Reply:
Or anywhere else in the docs in general